teencode-org/teencode
# #153024206 Add progress tracking template link in curriculum page

#### What does this PR do?
- Adds a link to the progress tracking template on the curriculum page

#### Description of Task to be completed?
- As a user, I can visit `'/curriculum'` and click on a `'Progress tracker template'` button/link to open a read-only template spreadsheet. Button should open spreadsheet in a new tab

#### How should this be manually tested?
- Log into the app
- Click on a curriculum header link to go to the curriculum page
- A `Progress Tracking Template (For facilitators)` link should be visible
- Click the link to see open the template in a new tab

#### What are the relevant pivotal tracker stories?
- #153024206

#### Any background context you want to add?
- N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-01-29 at 8 14 10 pm" src="https://user-images.githubusercontent.com/8125356/35529333-02bff8de-0531-11e8-8ea1-678727e5cce7.png">
